src/app/gauge/gauge.component.css

```diff
@@ -8,7 +8,7 @@
 
 .ng-gauge {
   position: relative;
-  width: 400px;
+  display: inline-block;
```

Comment:
Let's please keep the component as a `block` element. This can be set from outside of it with a `/deep/`, if the user desires it.